build/bin/install_tarball

```diff
@@ -73,3 +76,5 @@ if [ "${update}" = "1" ]; then
 else
   ${INSTALL_COMMAND} -m 0660 /dev/null ${PMM_DIR}/config/pmm-agent.yaml
 fi
+
+cd - || true
```

Comment:
better to save current directory and then just remove to it.
as in future there could be another `cd` command and we would easily return into incorrect directory